### ISSUE TITLE
require() triggers a warning if running in browser

### ISSUE THREAD

**Issue by tacman** [twigjs/twig.js#880]
There is a warning in then console about the require function not being present when running twigjs as an imported module.

Please see https://github.com/symfony/symfony/discussions/52252# 

The code still works, but it would be nice to get rid of this warning.

**Comment by RobLoach** [comment]
Looks like the loader could use some tweaks to have it function better with the browser:
https://github.com/twigjs/twig.js/blob/master/src/twig.loader.fs.js#L9

**Comment by tacman** [comment]
I used to have something in place when I used webpack to avoid the error, but I don't know if it will work with modules.  What is fs, and is there a package on jsdelivr that can be used for it?

**Comment by willrowe** [comment]
In what context is the fs loader being used in the browser? Wouldn't you use a different loader in that case?

**Comment by tacman** [comment]
as a module.  See the js console at
https://jsfiddle.net/tacman1123/azb4dmwe/7/

**Comment by willrowe** [comment]
Interesting, so it happens just by importing without doing anything else?

**Comment by tacman** [comment]
yes.  But fortunately it works (beautifully).  The code is wrapping in a
try / catch, but still throws the warning.
